app/controllers/folders_controller.rb

```diff
@@ -30,10 +30,13 @@ class FoldersController < ApplicationController
   before_action :admin_user, only: [:new, :create, :edit, :update, :destroy]
 
   def index
-    @folders = Folder.order(:name)
-
-    @folders.each do |a|
-      a.attachments.build
+    if params[:search]
```

Comment:
while it fundamentally does the same thing I like to throw a `.present?` on these just so it reads better

Reply:
sure i can add that!